### DISCUSSION TITLE
Update-DbaInstance Exiting with Warning

### DISCUSSION THREAD

**Discussion by ghost** [dataplat/dbatools#7904] [Q&A]
Hi,

I need help with the Update-DbaInstance cmdlet which is exiting with a warning message.

I'm trying to install cumulative updates on SQL Server 2019 instances on Windows Server 2016 using Update-DbaInstance.  I'm running version 1.1.12 of dbatools.  The servers I am targeting have CU11 installed and I am trying to install CU13.  The servers are not connected to the internet.

When I run Update-DbaInstance it returns the following warning,

"WARNING:  [12:31:58][Get-DbaBuildReferenceIndex] Index is stale, last update on:  2021-08-04T00:00:00, try the -Update parameter to fetch the most up to date index"

The commandlet Get-DbaBuildReferenceIndex doesn't exist in the version of dbatools I'm using (1.1.12) - or it's not publicly accessible - and I've tried running Get-DbaBuildReference before Update-DbaInstance but it returns the same warning message.  Running the command with the -Update parameter won't help because the servers are not connected to the internet.  I've included a screenshot of the script I'm running including the output.

Has anyone encountered this behaviour?  If so is there a workaround?  Or something else I need to check and/or enable?

Thank you.

Malcolm Swan

![image](https://user-images.githubusercontent.com/12680902/139373195-eafaa4b9-a473-4f98-9a7e-dfe68b76af33.png)

**Comment by andreasjordan** [comment]
You can download the build reference here: https://sqlcollaborative.github.io/assets/dbatools-buildref-index.json
The file has to be placed in the bin folder of the module.
